### ISSUE TITLE
GPT训练完文件夹没文件

### ISSUE THREAD

**Issue by HSJDZNM** [RVC-Boss/GPT-SoVITS#936]
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\pytorch_lightning\loops\fit_loop.py:293: The number of training batches (17) is smaller than the logging interval Trainer(log_every_n_steps=50). Set a lower value for log_every_n_steps if you want to see logs for the training epoch.
`Trainer.fit` stopped: `max_epochs=15` reached.
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\transformers\utils\generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\transformers\utils\generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\transformers\utils\generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\transformers\utils\generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
"C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\python.exe" GPT_SoVITS/s1_train.py --config_file "C:\Users\dell\Desktop\GPT-SoVITS-beta0306\TEMP/tmp_s1.yaml"
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\transformers\utils\generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
Seed set to 1234
Using 16bit Automatic Mixed Precision (AMP)
GPU available: True (cuda), used: True
TPU available: False, using: 0 TPU cores
IPU available: False, using: 0 IPUs
HPU available: False, using: 0 HPUs
<All keys matched successfully>
ckpt_path: logs\KoBy\logs_s1\ckpt\epoch=14-step=60.ckpt
[rank: 0] Seed set to 1234
Initializing distributed: GLOBAL_RANK: 0, MEMBER: 1/1
[W socket.cpp:697] [c10d] The client socket has failed to connect to [DESKTOP-FUHLA6V]:52405 (system error: 10049 - 在其上下文中，该请求的地址无效。).
----------------------------------------------------------------------------------------------------
distributed_backend=gloo
All distributed processes registered. Starting with 1 processes
----------------------------------------------------------------------------------------------------

semantic_data_len: 44
phoneme_data_len: 44
                                            item_name
                         semantic_audio
0   vocal_man.wav.reformatted.wav_10.wav_000002304...  520 486 486 486 486 486 486 486 486 486 72 704...
1   vocal_man.wav.reformatted.wav_10.wav_000069664...  774 647 774 953 565 192 747 973 940 940 526 67...
2   vocal_man.wav.reformatted.wav_10.wav_000131552...  1005 263 263 581 581 185 496 774 263 263 263 5...
& 语言文件........ &
deleted 34 audios who's phoneme/sec are bigger than 25 or smaller than 3
dataset.__len__(): 100
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\pytorch_lightning\callbacks\model_checkpoint.py:639: Checkpoint directory C:\Users\dell\Desktop\GPT-SoVITS-beta0306\logs\KoBy\logs_s1\ckpt exists and is not empty.
Restoring states from the checkpoint path at logs\KoBy\logs_s1\ckpt\epoch=14-step=60.ckpt
LOCAL_RANK: 0 - CUDA_VISIBLE_DEVICES: [0]

  | Name  | Type                 | Params
-----------------------------------------------
0 | model | Text2SemanticDecoder | 77.5 M
-----------------------------------------------
77.5 M    Trainable params
0         Non-trainable params
77.5 M    Total params
309.975   Total estimated model params size (MB)
Restored all states from the checkpoint at logs\KoBy\logs_s1\ckpt\epoch=14-step=60.ckpt
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\pytorch_lightning\loops\fit_loop.py:293: The number of training batches (17) is smaller than the logging interval Trainer(log_every_n_steps=50). Set a lower value for log_every_n_steps if you want to see logs for the training epoch.
`Trainer.fit` stopped: `max_epochs=15` reached.
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\transformers\utils\generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\transformers\utils\generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\transformers\utils\generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\transformers\utils\generic.py:441: UserWarning: torch.utils._pytree._register_pytree_node is deprecated. Please use torch.utils._pytree.register_pytree_node instead.
  _torch_pytree._register_pytree_node(

![屏幕截图 2024-04-07 192807](https://github.com/RVC-Boss/GPT-SoVITS/assets/131422417/d24e0d65-29f4-4d81-8868-b8d4e2ea2186)

**Comment by SapphireLab** [comment]
你这个...粘贴日志之前还是整理一下格式为好.

关键句:
```
C:\Users\dell\Desktop\GPT-SoVITS-beta0306\runtime\lib\site-packages\pytorch_lightning\callbacks\model_checkpoint.py:639:
Checkpoint directory C:\Users\dell\Desktop\GPT-SoVITS-beta0306\logs\KoBy\logs_s1\ckpt exists and is not empty.
Restoring states from the checkpoint path at logs\KoBy\logs_s1\ckpt\epoch=14-step=60.ckpt
```

看起来是之前用实验名 `KoBy` 训练过, 并且 `logs` 下的文件夹没有清除, 读取到了相应模型, 其训练的 epoch 数已经达到上限, 所以没有继续训练.
请考虑清除 `logs/KoBy` 文件夹或者重新设置实验名.
